Billow develop

drower opened the pull request:
add feature Content/Advertise

drower committed:
add feature Content/Advertise(admin);

drower committed:
add feature Content/Mods/AD

drower committed:
update Content/Advertise(admin) link,order,style

drower committed:
update Content/Mods/AD style

wangxiaomo commented on MinMore/Application/Content/Controller/AdvertiseController.class.php:
```diff
@@ -0,0 +1,102 @@
+
+	//广告位列表
+	public function index() {
+		$data = M('Advertise')->order(array('status'=>'DESC','type'=>'ASC','id' => 'DESC'))->select();
```
取的时候也需要根据roleid来取吧？

drower replied:
嗯，前台是加了的，后台还特地在初始化时就取了site_role了，竟然在后台列表这里漏掉了这个条件。。

wangxiaomo commented on MinMore/Application/Content/Controller/IndexController.class.php:
```diff
@@ -330,5 +330,29 @@ public function tags() {
         $this->display("Tags/tag");
     }
+	
+    public function advertise() {
```
因为侧边栏广告是每个页面都要共用的，那么取数据的这部分是不是应该放到更高层级的controller里面？MinMoreController

wangxiaomo commented on MinMore/Template/L1_Global/Content/Index/index.php:
```diff
@@ -458,6 +458,7 @@ function(){
         <template file="Content/Mods/footer.php" />
     </div>
     <template file="Content/Mods/QR.php"/>
+    <template file="Content/Mods/AD.php"/>
```
不止首页，一级站点、二级站点所有的首页、内容页、列表页、单页，都需要引入这个mod.

drower committed:
fix Content/Advertise/index bind with site_role(admin)

drower committed:
1.update  Content/Index/advertise move to MinmoreCMS as a Global interface;
2.update set advertise as a public Mod for L1_Global and L2_Global;
3.fix style confilict when AD Mod publish to L2_Global(class link in common.css);

drower committed:
fix Content/Index/index.php duplicate import AD mod

drower committed:
fix Content/Index/index.php footer position

drower committed:
fix remove advertise from /Content/Index
